e2e/test/scenarios/admin-2/settings.cy.spec.js

```diff
@@ -548,9 +548,12 @@ describe("scenarios > admin > settings > email settings", () => {
       Cypress.config().baseUrl + "/admin/settings/email",
     );
     cy.findByTestId("smtp-connection-card").should("exist");
-
     // Non SMTP-settings should save automatically
-    cy.findByLabelText("From Address").type("mailer@metabase.test").blur();
+    cy.findByLabelText("From Address")
+      .clear()
```

Comment:
Note that I think the AdminSettingInput component correctly reads default settings values, but the old settings components didn't. This test expected the input to initially be empty, relying on the old behaviour

Reply:
I think this is because were now always getting settings values from session/properties, instead of sometimes from the /settings array